UnitySDK/Assets/ML-Agents/Scripts/IBrain.cs

```diff
@@ -0,0 +1,23 @@
+using System;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace MLAgents
+{
+    /// <summary>
+    /// Brain receive data from Agents through calls to SubscribeAgentForDecision. The brain then updates the
```

Comment:
The terrible name `SubscribeAgentForDecision` has thankfully been changed to `RequestDecision`.